src/Silk.Data/EntityMapping/EntityMapping.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Mapster;
+using Silk.Data.DTOs.Guilds.Config;
+using Silk.Data.Entities;
+using Silk.Data.Entities.Guild.Config;
+
+namespace Silk.Data.EntityMapping;
+
+// Most likely move mapping to another common library
+public static class EntityMapping
+{
+    public static void ConfigureMappings()
```

Comment:
Semicolons should reside on the same line as the closing parentheses; furthermore, method chaining should aline with the furthermost outdent.

Reply:
👍forgot to move them back, was easier having them underneath when needing to add more `.Map` calls.